roles/metrics/templates/metrics.sh.j2

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+# -------------------------------
+# Configuration
+# -------------------------------
+VALOPER="{{ vars[project].validator.valoper_address }}"
+APIS=(
+{% for url in vars[project].validator.api_urls %}
+  "{{ url }}"
+{% endfor %}
+)
+
+PUSHGATEWAY_URL="https://{{ pushgateway_dns }}"
+JOB_NAME="chain_exporter"
+PROJECT="{{ project }}"
+ENV="{{ env }}"
+AUTH_HEADER="Authorization: Basic {{ vault.default.pushgateway.auth_header }}"
+TIMEOUT=30
+
+# -------------------------------
+# Timestamped logger
+# -------------------------------
+log() {
+  echo "[$(date '+%Y-%m-%d %H:%M:%S')] $1"
+}
+
+# -------------------------------
+# Function: Check jailed status
+# -------------------------------
+check_jailed() {
+  local api="$1"
+  local url="${api}/cosmos/staking/v1beta1/validators/${VALOPER}"
+  local response
+  response=$(curl -s --max-time 5 "$url")
+
+  if [[ -z "$response" ]]; then
+    return 1
+  fi
+
+  local jailed
+  jailed=$(echo "$response" | jq -r '.validator.jailed' 2>/dev/null)
+
+  if [[ "$jailed" == "true" ]]; then
+    echo "jailed"
+    return 1
+  elif [[ "$jailed" == "false" ]]; then
+    echo "not jailed"
+    return 0
+  else
+    return 1
+  fi
+}
+
+# -------------------------------
+# Main logic
+# -------------------------------
+log "Checking jailed status for validator..."
+
+status=""
+exit_code=2
+api_success=false
+
+for api in "${APIS[@]}"; do
+  result=$(check_jailed "$api")
+  if [[ $? -eq 0 ]]; then
+    status="not jailed"
+    exit_code=0
+    api_success=true
+    break
+  elif [[ "$result" == "jailed" ]]; then
```

Comment:
please make the if else consistent, if we are checking on return code, then lets do that or the else too

in if we have $? -eq 0 but in else we $result" == "jailed"